internal/config/config.go

```diff
@@ -63,6 +63,9 @@ type Config struct {
 	ciVisibilityAgentless         bool
 	logDirectory                  string
 	traceRateLimitPerSecond       float64
+	// universalVersion, reports whether span service name and config service name
+	// should match to set application version tag.
```

Comment:
```suggestion
	// universalVersion, defines the version to attach to every span. When empty the version is attached only when the span service matches the global service name.
```

The current comment does not really tell what it is and used for.